Makefile

```diff
@@ -28,7 +28,6 @@ integration-test-py3: build-py3
 	docker run -v `$(HOST_TMPDIR)`:/tmp -v /var/run/docker.sock:/var/run/docker.sock docker-py3 py.test -rxs tests/integration_test.py
 
 integration-dind: build build-py3
-	docker build -t dpy-tests -f ./tests/Dockerfile .
```

Comment:
No, it's just unneeded, and I forgot to remove it - we already have Dockerfiles for building docker-py at the root.